src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,77 @@ user_questions:
   - What is the architecture of the Giant Swarm cloud-native developer platform?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-planeteers
-last_review_date: 2024-04-26
+last_review_date: 2024-11-04
 ---
 
-## Platform API {#platform-api}
+Giant Swarm's cloud-native platform is a collection of open-source components that work together to provide a seamless experience for managing the lifecycle of containerized applications. The platform is designed to be cloud-agnostic, allowing you to deploy your applications on any of the supported cloud providers. Besides, the platform offers a rich set of APIs that our customers can use to offer a set of self-service capabilities to their developers.
```

Comment:
```suggestion
Giant Swarm's cloud-native developer platform integrates open-source components that work together to provide a seamless experience for managing the lifecycle of containerized applications. The platform is based on Kubernetes and designed to be cloud-agnostic, allowing you to deploy your applications on any of the supported cloud providers, including on-premises. The interfaces allow development teams and automations to deploy applications, keep the clusters secure, and use other capabilities that we explain below. On top, you benefit from our strong support model that ensures your long-term success and stability of workloads.
```

Changes I made:

- Don't use the word "collection" since it sounds too much like the platform isn't adding much value and just puts pieces together
- Replaced "Rich set of APIs" since that's not a selling point. Less is more. We should rather keep it short here, and explain all interfaces below.
- Added a short note of the support model, just to have mentioned each part of the diagram also in the text.